components/31-molecules/mijn-gent-block/_mijn-gent-block.scss

```diff
@@ -47,6 +60,12 @@
     &::before {
       line-height: 0;
     }
+
+    span.avatar {
+      display: block;
+      width: 2.3rem;
+      height: 2.3rem;
```

Comment:
`width` and `height` are inherited from https://github.com/StadGent/fractal_styleguide_gent-base/blob/eedd20b26689093c5f91498ecf835983f8856812/components/31-molecules/mijn-gent-block/_mijn-gent-block.scss#L6-L18

and could be removed